proofs.go

```diff
@@ -34,10 +34,11 @@ func createChallenge(context, nonce *big.Int, contributions []*big.Int, issig bo
 // ProofU represents a proof of correctness of the commitment in the first phase
 // of the issuance protocol.
 type ProofU struct {
-	U              *big.Int `json:"U"`
-	C              *big.Int `json:"c"`
-	VPrimeResponse *big.Int `json:"v_prime_response"`
-	SResponse      *big.Int `json:"s_response"`
+	U              *big.Int         `json:"U"`
+	C              *big.Int         `json:"c"`
+	VPrimeResponse *big.Int         `json:"v_prime_response"`
+	SResponse      *big.Int         `json:"s_response"`
+	MUserResponses map[int]*big.Int `json:"m_user_responses"`
```

Comment:
Please add `,omitempty` to avoid `m_user_responses` from being included in the JSON if it is empty/nil.

Reply:
Good catch, as most of the time this will be empty for most `Proofs`/`IssueSignatureMessages`